buildinggame/src/main/java/com/gmail/stefvanschiedev/buildinggame/Main.java

```diff
@@ -426,4 +432,23 @@ public void loadPlugin(boolean reload) {
     public static Main getInstance() {
 		return instance;
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Returns false if the user using old java version, that is not supports.
+	 * @return boolean
+	*/
+	private boolean checkJavaVersion() {
+		try {
+			// Java versions: https://en.wikipedia.org/wiki/Java_class_file
+			if (Float.parseFloat(StandardSystemProperty.JAVA_CLASS_VERSION.value()) < 55.0) {
+				getLogger().warning("You are using an older Java that is not supported. Please use 1.11 or higher versions!");
```

Comment:
This method should probably not log things itself, but the caller should decide what to do with the returned result. That way this method doesn't have any side effects.